### DISCUSSION TITLE
The game resume playing the song in a wrong time when back to beatmap select by score page

### DISCUSSION THREAD

**Discussion by xiaoniaoawa** [ppy/osu#30167] [Q&A]
### Type

Game behaviour

### Bug description

Before issue: My English is not good, if any question, ask for me.

The default play point of this map:
![image](https://github.com/user-attachments/assets/0e321bf8-c52a-43a8-88ef-cc0e62534a9a)

0 - pause a song in beatmap select page
![image](https://github.com/user-attachments/assets/5b146461-e68c-47a3-a6c8-fdc1efa10621)
![image](https://github.com/user-attachments/assets/99d65841-f724-4020-a794-6e7f753c7228)

1 - Open a score page
![image](https://github.com/user-attachments/assets/d84b1a7e-209d-47b2-8d19-77bfb71fd0bb)
2 - back
3 - You seen the song resume in the pause point, not the default point
![image](https://github.com/user-attachments/assets/6a7bdf33-aa4a-41ca-87ff-846d0de3ac3b)

### Screenshots or videos

How to: https://www.bilibili.com/video/BV1g4xTefEwP/

### Version

Lazer 2024.906.2

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/17174356/compressed-logs.zip)

**Comment by KamuiSW** [comment]
maybe it's an intentional feature? just a guess :D

**Comment by smoogipoo** [comment]
I would say this is intentional. The beatmap hasn't changed so there's no reason the song would be restarted.
